github/actions_secrets.go

```diff
@@ -66,42 +62,44 @@ func (s *ActionsService) GetRepoPublicKey(ctx context.Context, owner, repo strin
 	return pubKey, resp, nil
 }
 
+// GetRepoPublicKey gets a public key that should be used for secret encryption.
+//
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#get-a-repository-public-key
+func (s *ActionsService) GetRepoPublicKey(ctx context.Context, owner, repo string) (*PublicKey, *Response, error) {
+	url := fmt.Sprintf("repos/%v/%v/actions/secrets/public-key", owner, repo)
+	return s.getPublicKey(ctx, url)
+}
+
+// GetRepoDependabotPublicKey gets a public key that should be used for secret encryption.
+//
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#get-a-repository-public-key
+func (s *ActionsService) GetRepoDependabotPublicKey(ctx context.Context, owner, repo string) (*PublicKey, *Response, error) {
+	url := fmt.Sprintf("repos/%v/%v/dependabot/secrets/public-key", owner, repo)
+	return s.getPublicKey(ctx, url)
+}
+
 // GetOrgPublicKey gets a public key that should be used for secret encryption.
 //
 // GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#get-an-organization-public-key
 func (s *ActionsService) GetOrgPublicKey(ctx context.Context, org string) (*PublicKey, *Response, error) {
-	u := fmt.Sprintf("orgs/%v/actions/secrets/public-key", org)
-	req, err := s.client.NewRequest("GET", u, nil)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	pubKey := new(PublicKey)
-	resp, err := s.client.Do(ctx, req, pubKey)
-	if err != nil {
-		return nil, resp, err
-	}
+	url := fmt.Sprintf("orgs/%v/actions/secrets/public-key", org)
+	return s.getPublicKey(ctx, url)
+}
 
-	return pubKey, resp, nil
+// GetOrgDependabotPublicKey gets a public key that should be used for secret encryption.
+//
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#get-an-organization-public-key
```

Comment:
Is there a better URL for this endpoint?

Reply:
https://docs.github.com/en/rest/reference/dependabot#get-an-organization-public-key